tools/grunt-tasks/qunit.js

```diff
@@ -0,0 +1,465 @@
+/*
+ * grunt-contrib-qunit
```

Comment:
Copied from grunt-contrib-quint to remove the dependencies to directly fix the governance issues from this version to limit the amount of required changes

Reply:
This file has some changes which include 
- updates location of the bridge.js file
- directly includes the imported `pEachSeries` rather than use as a dependency
- Some minor fixes around the usage of the error.stack
- Merge some minor fixes from later releases to this version (required as we are still building in node 16 and the version of qunit we are using)